tests/async-unit/test_ml_router.py

```
@@ -0,0 +1,453 @@
+        history = [
+            {"role": "user", "content": long_content},
+            {"role": "assistant", "content": "Short response"},
+        ] * 8  # Make it long enough to trigger summarization
+
+        # Create a mock model
+        mock_model = AsyncMock(spec=RemoteModelLike)
+        mock_model._infer_no_context.return_value = "Summary with truncated messages"
+
+        self.router.internal_models_by_cost = [mock_model]
+
+        result = await self.router.summarize_chat_history(
```

Ruff (0.14.10)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)